### ISSUE TITLE
[TASK] Replica rebuilding performance measurement

### ISSUE THREAD

**Issue by innobead** [longhorn/longhorn#4234]
## What's the task? Please describe

Replica rebuilding is a frequent activity for each volume to ensure volume availability.  It's better to measure the performance of replica rebuilding based on different impacted factors to know how much resource cost will be required for each node. This can help us understand any potential impacts (direct or indirect/cascading) on the system, then indirectly impact Longhorn operations. 

Also, we have `Concurrent Replica Rebuild Per Node Limit` to control the number of replicas to rebuild per node, but having a performance reference to understand the cost behind it will better understand how to set the setting meaningfully. 

## Describe the items of the task (DoD, definition of done) you'd like

- [ ] Measure the CPU and memory consumption for 1-n replicas (need to think about some correlation between factors)

## Additional context

N/A

**Comment by joshimoo** [comment]
Just leaving a note here in regard to a major contributor to rebuild performance.
On older kernels and with ext4 fs for the longhorn storage mount, rebuild performance will be slow.
See https://github.com/longhorn/longhorn/issues/2640 for more details.

TLDR:
`5.4.0-73-generic = slow, 5.8.0-53-generic = fast`
